### PR TITLE
route53 records

### PR DESCRIPTION
**Pull request by Baggerone**
Inspired by
https://github.com/alemuro/terraform-aws-ses-email-forwarding/

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -92,9 +92,37 @@ resource "aws_lambda_function" "ses_redirecter" {
...
+
+  count = 1
+}
 
 data "template_file" "instructions" {
-  template = file("${path.module}/instructions.txt")
+  template = file("${path.module}/instructions.md")
 
   vars = {
     aws_region = var.aws_region
```
Comment (briskt) [inline main.tf]: `aws_region` isn't in the template file. Do you want to keep it here?

**Review by briskt — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (fillup) [reply main.tf]: hmm, it's nice to support Route53 if someone else needs that, but we don't use Route53 so this wouldn't work for us. We need to use Cloudflare. So either we add support to use either depending on parameters sent to it, or maybe have a base model and one or two extended modules that make use of the base but then create dns records in the appropriate place

Comment (Baggerone) [reply main.tf]: Oh.  When I was doing it manually, I did it through Route53.

File: outputs.tf [hunk 2]
```diff
@@ -2,6 +2,11 @@ output "function_arn" {
   value = aws_lambda_function.ses_redirecter.arn
 }
 
+output "record_mx_record" {
+  value = join(" ", [aws_route53_record.mx.name, aws_route53_record.mx.records[0]])
```
Comment (fillup) [inline outputs.tf]: this appears to still reference route53. I think the records needed would come from an ses resource right?

Comment (Baggerone) [reply outputs.tf]: Oops. I didn't re-look at that one.

**Pull request merged**